src/main/java/com/vinylteam/vinyl/util/ShopsParser.java

```diff
@@ -15,10 +19,12 @@ public ShopsParser(List<VinylParser> vinylParserList) {
 
     public List<Vinyl> getAllVinyls() throws IOException {
         List<Vinyl> allVinyls = new ArrayList<>();
-
         for (VinylParser vinylParser : vinylParserList) {
             allVinyls.addAll(vinylParser.getAllVinylsFromShopList());
+            logger.debug("Added all vinyls from vinylParser.getAllVinylsFromShopList() " +
+                    "to list of all vinyls {'vinylParser':{}, 'allVinyls':{}}", vinylParser, allVinyls);
```

Comment:
logger.debug("Added all vinyls to all vinyls list: {''allVinyls':{}}", allVinyls) or better remove it at all. We will have all vinyls list in the 27 row